docs/source/modules/convection_simple_betts_miller.rst

```diff
@@ -0,0 +1,95 @@
+Convection scheme: Simple Betts-Miller
+=================================
+
+
+Summary
+-------
+The simplified Betts-Miller (SBM) scheme is a moist adjustment convection scheme developed by [Frierson2007]_ and the modifications described in [OGormanSchneider2008]_. The SBM scheme is Isca's default convection scheme. The key purpose of the routine is to output a moisture tendency ``deltaq``, a temperature tendency ``deltaT``, and a convective rainfall amount ``rain``. The scheme has both deep convection and non-precipitating shallow convection.
+
+
+Logic of the routine
+-------
+
+1. Perform a particle ascent to calculate the environmental vertical profiles of moisture and temperature, and convective available potential energy (CAPE) within a grid cell.
+
+2. Test if the convection scheme should trigger. The trigger is: :math:`CAPE>0`.
```

Comment:
Just a question: Does this mean the scheme should turn on or that is is triggering someting?

Reply:
Good point. I will update.